src/logging/mod.rs

```diff
@@ -0,0 +1,162 @@
+/// Logging via slog
```

Comment:
Can you add the MPL preamble comment to the top of this module? Just copy it from one of the others...